gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/PipelineController.groovy

```diff
@@ -60,10 +60,39 @@ class PipelineController {
   @Autowired
   ObjectMapper objectMapper
 
+  @CompileDynamic
```

Comment:
Why is this needed?

Reply:
This method has some dynamic properties added in the result at line no. 84....we will checking for result properties for exception, hence the method has to be made CompileDynamic.
Same will be the case for savePipeline() method.